lib/restful_model_collection.rb

```diff
@@ -37,6 +37,9 @@ def count
     end
 
     def first
+      @filters[:offset] = 0
```

Comment:
I just added a test (https://github.com/nylas/nylas-ruby/pull/74/files#diff-ebbc5b209198fb9bc40897cc756c8262R212) to make sure that isn't the case, and it seems like we're good.